tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.robot

```diff
@@ -0,0 +1,78 @@
+*** Settings ***
+Documentation    Collection of keywords to interact with Model Servers
+Resource       ../../../../Page/Components/Components.resource
+Resource       ../../../../Common.robot
+Resource       Projects.resource
+
+
+*** Variables ***
+${S3_NAME_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field Name"]
+${S3_KEY_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field AWS_ACCESS_KEY_ID"]
+${S3_SECRET_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field AWS_SECRET_ACCESS_KEY"]
+${S3_ENDPOINT_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field AWS_S3_ENDPOINT"]
+${S3_REGION_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field AWS_DEFAULT_REGION"]
+${DC_SECTION_XP}=             xpath=//div[div//h4[@id="data-connections"]]
+${DC_ADD_BTN_1_XP}=           ${DC_SECTION_XP}/div/button[text()="Add data connection"]
+${DC_ADD_BTN_2_XP}=           xpath=//footer/button[text()="Add data connection"]
+${S3_BUCKET_DC_INPUT_XP}=     xpath=//input[@aria-label="AWS field AWS_S3_BUCKET"]
+${S3_DEFAULT_ENDPOINT}=    https://s3.amazonaws.com/
+${S3_DEFAULT_REGION}=    us-east-1
```

Comment:
would it make sense to move these variables in the test-variables.yaml along with other S3 data which are already in there?

Reply:
Yes, will do that